<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNew</class>
 <widget class="QDialog" name="addNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="push_addTb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_cancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Referência</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ln_ref"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>201</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Proa: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_proa"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Distância</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Parcial:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_dParc"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Acumulada:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_dAc"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>295</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Hora estimada de passagem:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_hEst"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Hora real de passagem:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_hRe"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Tempo gasto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_horaReal"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Velocidade resultante:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_velRes"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>208</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Temp. calculado:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Parcial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_tempParc"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Acumulado:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_tempAc"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>171</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Situação</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_sit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="lb_dist">
        <property name="text">
         <string>Distância: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_sitDist"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
